api/entity_sensors/sensors.go

```diff
@@ -11,7 +11,9 @@ import (
 
 func SensorValue(network *net.Network, application, entity, sensor string) (string, error) {
 	url := fmt.Sprintf("/v1/applications/%s/entities/%s/sensors/%s", application, entity, sensor)
-	body, err := network.SendGetRequest(url)
+	headers := make(map[string]string)
+	headers["Accept"] = "text/plain"
+	body, err := network.SendGetRequestWithHeaders(url, headers)
```

Comment:
For the sake of consistency, I'd use `json.Unmarshal` instead of sending different headers, see e.g. `SensorList`.